modules/apigee_graphql_doc/src/Plugin/Field/FieldFormatter/GraphqlFormatter.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+/**
+ * @file
+ * Copyright 2021 Google Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License version 2 as published by the
+ * Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc., 51
+ * Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ */
+
+namespace Drupal\apigee_graphql_doc\Plugin\Field\FieldFormatter;
+
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\Core\Field\FormatterBase;
+
+/**
+ * Plugin implementation of the 'Graphql' formatter.
+ *
+ * @FieldFormatter(
+ *   id = "apigee_graphql_doc_graphql",
+ *   label = @Translation("GraphQL"),
+ *   field_types = {
+ *     "file_link"
+ *   }
+ * )
+ */
+class GraphqlFormatter extends FormatterBase {
```

Comment:
Should we call this formatter GraphiQL ? Since it is based on the graphiql library. ?